Patch by Martin Vaeth <martin at mvath.de> from Oct 3, 2016
--- 1/ft.c
+++ 1/ft.c
@@ -13,11 +13,11 @@
 #include <ctype.h>
 #include <sys/types.h>
 #include <ft2build.h>
-#include <freetype2/freetype.h>
-#include <freetype2/ftglyph.h>
-#include <freetype2/ftsnames.h>
-#include <freetype2/ttnameid.h>
-#include <freetype2/ftoutln.h>
+#include <freetype/freetype.h>
+#include <freetype/ftglyph.h>
+#include <freetype/ftsnames.h>
+#include <freetype/ttnameid.h>
+#include <freetype/ftoutln.h>
 #include "pt1.h"
 #include "global.h"
 
